lamp-stack-ubuntu/README.md

```diff
@@ -0,0 +1,6 @@
+# Lamp stack on Ubuntu VM
```

Comment:
A similar template already exists https://github.com/Azure/azure-quickstart-templates/tree/master/lamp-app

Does this template offer anything over the existing template?